molgenis-charts/src/main/java/org/molgenis/charts/highcharts/convert/HighchartSeriesUtil.java

```diff
@@ -137,45 +142,4 @@ else if (AttributeType.DATE.equals(fieldTypeEnum))
 		}
 	}
 
-	/**
-	 * Convert date to long keeping the timezone valued date. When asking the time of a Date object java return the
-	 * milliseconds from the begin of counting. "Returns the number of milliseconds since January 1, 1970, 00:00:00 GMT
-	 * represented by this Date object."
-	 * <p>
-	 * This can be a problem when accepting JavaSript to create a JavaScript Date object not knowing the time zone and
-	 * ..
-	 */
-	public Long convertDateToMilliseconds(Date date)
-	{
-		if (date == null) return null;
-		Calendar calendar = Calendar.getInstance();
-		calendar.setTime(date);
-		Calendar calendarConverted = Calendar.getInstance(TimeZone.getTimeZone("GMT"), Locale.ENGLISH);
-		calendarConverted.clear();
-		calendarConverted.set(calendar.get(Calendar.YEAR), calendar.get(Calendar.MONTH), calendar.get(Calendar.DATE));
-		return calendarConverted.getTimeInMillis();
```

Comment:
This code is a symptom of the problem. A java.util.Date is a wrapper around a long and represents a moment in time. If you use java.util.Date to store a local date without time, you need to convert the local date to a moment in time. High charts expects a date value to be transferred as epoch millis on the moment that that the day of that date starts in UTC. But molgenis internally used time zone Europe/Amsterdam. Here we convert from the one timezone to the other. Now that we use Instant molgenis-wide, we can simply turn the Instant into epoch millis and be done with it.